### DISCUSSION TITLE
ONCE (one shot) clips: engage them, they play on the next synced moment with rest of loops, and then automute themselves

### DISCUSSION THREAD

**Discussion by soymonitus** [SynthstromAudible/DelugeFirmware#860] [Ideas]
Pretty much like FILL clips but synced to the beginning instead of to the end. It should be pretty straightforward to implement

**Comment by soymonitus** [comment]
When selected the type of clip, it could cycle between DEFAULT, ONCE and FILL

**Comment by Pluppo** [comment]
I like this idea a lot

**Comment by darenager** [comment]
Plus one for this, also FIRST (or 1st) trig condition - only plays event(s) on first pass through loop without need of user interaction. The difference being it can only be applied to chosen events, rather than at clip level as the proposed ONCE clip idea.

**Comment by trappar** [comment]
What's the difference between this and just setting the clip to only repeat once? (existing feature)

**Reply by soymonitus** [reply]
The existing repeat count will mute the current section and proceed playing the next section. Quite different
